lib/routes/account.js

```diff
@@ -1530,6 +1596,84 @@ module.exports = function (
     },
     {
       method: 'POST',
+      path: '/account/login/send_unblock_code',
```

Comment:
In practice, the customs server should rate limit the sending of an email (so rate limit this route per email address), and that combined with the lifetime of unblock codes, there should only be a few possible to use at most.